XYZDrivers/src/main/webapp/admin.jsp

```diff
@@ -30,65 +30,106 @@
+                    </div>
+                    <div class="panel-body">
+                        <p>ID : <c:out value="${provisionalMember.id}"/></p>
+                        <p>Name : <c:out value="${provisionalMember.name}"/></p>
+                        <p>Address : <c:out value="${provisionalMember.address}"/></p>
+                        <p>Date of Birth : <c:out value="${provisionalMember.dob}"/></p>
+                        <p>Date of Record : <c:out value="${provisionalMember.dor}"/></p>
+                        <p>Status : <c:out value="${provisionalMember.status}"/></p>
+                        <p>
+                            <a href="/XYZDrivers/membership-status?status=APPROVED&amp;member_id=<c:out value="${member.id}" />">
+                                Approve
+                            </a> /
+                            <a href="/XYZDrivers/membership-status?status=REJECTED&amp;member_id=<c:out value="${member.id}" />">
+                                Reject
+                            </a>
+                        </p>
+                    </div>
                 </div>
-                <div class="panel-body">
-                    <ul>
-                    <c:forEach items="${members}" var="member">
-                        <li>
-                            <c:out value="${member.id}"/>
-                        - name: <c:out value="${member.name}"/>
-                        - address: <c:out value="${member.address}"/>
-                        - dob: <c:out value="${member.dob}"/>
-                        - dor: <c:out value="${member.dor}"/>
-                        - status: <c:out value="${member.status}"/>
-                        - balance: <c:out value="${member.balance}"/>
-                        </li>
-                    </c:forEach>
-                    </ul>
+                </c:forEach>
+            </div>
+            <div class="col-md-4">
+                <h2>Members</h2>
+                <c:forEach items="${members}" var="member">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h3 class="panel-title"><c:out value="${member.name}"/></h3>
+                    </div>
+                    <div class="panel-body">
+                        <p>ID : <c:out value="${member.id}"/></p>
+                        <p>Name : <c:out value="${member.name}"/></p>
+                        <p>Address : <c:out value="${member.address}"/></p>
+                        <p>Date of Birth : <c:out value="${member.dob}"/></p>
+                        <p>Date of Record : <c:out value="${member.dor}"/></p>
+                        <p>Status : <c:out value="${member.status}"/></p>
+                        <p>Balance : <c:out value="${member.balance}"/></p>
+                        <p>
+                            <c:if test="${(member.status == 'APPROVED') or (member.status == 'REJECTED')}">
```

Comment:
Does rejected need to be added to the allowedStatuses ArrayList in the Member model?